vlog4j-parser/src/main/java/org/semanticweb/vlog4j/parser/javacc/JavaCCParserBase.java

```diff
@@ -230,4 +243,19 @@ public KnowledgeBase getKnowledgeBase() {
 		return knowledgeBase;
 	}
 
+	public void setParserConfiguration(ParserConfiguration parserConfiguration) {
```

Comment:
I would just make fields private, especially if we have getters and setters for them.

Reply:
I'm leaving them protected for now, since they are modified in the generated Parser, but this should be taken into account when refactoring the Parser.